frontend/src/views/project/UserWorkspaceList.js

```diff
@@ -2,13 +2,13 @@ import React, { useRef, useState } from 'react'
 import { Button, Tooltip, ButtonGroup, Menu, MenuItem } from '@material-ui/core'
 import { ArrowDropDown as ArrowDropDownIcon } from '@material-ui/icons'
 import { useMutation } from '@apollo/react-hooks'
-
+import { PROJECT_BY_ID } from '../../graphql/Query'
```

Comment:
Seems to be missing a blank line

```suggestion

import { PROJECT_BY_ID } from '../../graphql/Query'
```